### PR TITLE
[12.0][ADD] Withholding Tax Report

### PR DESCRIPTION
**Pull request by kongrattapong**
@kittiu

### CODE REVIEW

File: l10n_th_withholding_tax_report/wizard/withholding_tax_report_wizard.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+    def _export(self, report_type):
+        """Default export is PDF."""
+        type_excel = self._context.get('type', False)
+        model = self.env['withholding.tax.report']
+        report = model.create(self._prepare_wt_report())
+        report._compute_results()
+        if not type_excel:
+            return report.print_report(report_type)
+        print(report.env.ref('l10n_th_withholding_tax_report.report_withholding_txt_template'))
```
Comment (Saran440) [inline l10n_th_withholding_tax_report/wizard/withholding_tax_report_wizard.py]:
```suggestion
```

File: l10n_th_withholding_tax_report/wizard/withholding_tax_report_wizard.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        return report.env.ref('l10n_th_withholding_tax_report.report_withholding_txt_template')
```
Comment (Saran440) [inline l10n_th_withholding_tax_report/wizard/withholding_tax_report_wizard.py]:
```suggestion
        return report.env.ref('l10n_th_withholding_tax_report.report_withholding_txt_template')
```
line too long > 79 characters

**Review by Saran440 — COMMENTED** [review]
Code Reviewed

**Comment by Saran440** [comment]
I can't install this module. This error
```
odoo.tools.convert.ParseError: "Wrong value for ir.actions.report.report_type: 'excel'" while parsing /home/saran/venv_odoo12/workspace/rjc/l10n_th_withholding_tax_report/report_txt/report.xml:9, near
<report id="action_report_withholding_tax_txt" string="Withholding Tax Report (.txt)" model="withholding.tax.report" name="report_withholding_tax_txt.xlsx" file="ReportPNDTemplate" report_type="excel"/>
```

**Comment by kittiu** [comment]
@Saran440 I think Kong still stuck on the new excel_import_excel, which I have transfer this module to me to continue. I will tell you when it is ready for review.

**Comment by Saran440** [comment]
@kittiu  okay, I see.

**Comment by kittiu** [comment]
@kongrattapong 
there is warning caused by, https://github.com/OCA/l10n-thailand/pull/43, once it is merged, please rebase this and we can merge when this is green. Thanks!

**Pull request closed without merging**